templates/conference/index.html.twig

```diff
@@ -13,6 +13,7 @@
                 <th>Description</th>
                 <th>Date</th>
                 <th>actions</th>
```

Comment:
I suggest to display note before actions

Reply:
good suggestion